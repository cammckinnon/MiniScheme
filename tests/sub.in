(- 5 3)
(- 15 4 3 2)
(- 18)
(- 6 4 4)