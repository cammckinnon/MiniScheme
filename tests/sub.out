2.0
6.0
-18.0
-2.0